cds_ils/importer/providers/ebl/rules/document.py

```diff
@@ -125,6 +129,14 @@ def identifiers(self, key, value):
         }
         if isbn not in _identifiers:
             _identifiers.append(isbn)
+    if "q" in value and _eitem_type == "audiobook":
+        isbn = {
+            "scheme": "ISBN",
+            "value": clean_val("q", value, str, req=True),
```

Comment:
to be removed, discussed IRL: EBL will not have the audiobooks. also value of subfield q should not be stored in "value" but in "material"